packages/@aws-cdk/aws-lambda-nodejs/README.md

```diff
@@ -101,5 +101,24 @@ new lambda.NodejsFunction(this, 'my-handler', {
 
 The modules listed in `nodeModules` must be present in the `package.json`'s dependencies. The
 same version will be used for installation. If a lock file is detected (`package-lock.json` or
-`yarn.lock`) it will be used along with the right installer (`npm` or `yarn`). The modules are
-installed in a [Lambda compatible Docker container](https://hub.docker.com/r/amazon/aws-sam-cli-build-image-nodejs12.x).
+`yarn.lock`) it will be used along with the right installer (`npm` or `yarn`).
+
+
+### Local bundling
+If Parcel v2 is available it will be used to bundle your code in your environment. Otherwise,
```

Comment:
Should we be openly transparent and say something like "If you're on macOS, you should probably use this for performance reasons"?

Thanks!

Reply:
Good idea